acceptancetests/deploy_stack.py

```diff
@@ -1070,7 +1070,11 @@ def dump_all_logs(self, patch_dir=None):
                     runtime_config = None
                 artifacts_dir = os.path.join(self.log_dir,
                                              client.env.environment)
-                os.makedirs(artifacts_dir)
+                if not os.path.exists(artifacts_dir):
```

Comment:
When using try/except there is no need for this check, we try and make the dir and if it fails due to already existing that's fine ignore the exception.